robots/pkg/kubevirtci/kubevirtci.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 var SemVerRegex = regexp.MustCompile(`^[v]?([0-9]+)\.([0-9]+)\.([0-9]+)$`)
-var SemVerMinorRegex = regexp.MustCompile(`^([0-9]+)\.([0-9]+)$`)
+var SemVerMinorRegex = regexp.MustCompile(`^([0-9]+)\.([0-9]+)*`)
```

Comment:
I think this change hides what is really going on here. We should make the possible extension of the path name explicit like so

```suggestion
var SemVerMinorRegex = regexp.MustCompile(`^([0-9]+)\.([0-9]+)(-[a-z0-9-]+)?$`)
```

While we are at this change, we should also rename the `var`  

```suggestion
var ProviderFolderRegex = regexp.MustCompile(`^([0-9]+)\.([0-9]+)*`)
```

I can't remember out of my head, but isn't this regex also used in other places? Maybe we need to create a separate one for `isProviderDirectory` ?

Reply:
I made these updates here - https://github.com/kubevirt/project-infra/pull/2628/commits/cf91037faad3869f4cb554ee300f61aff3ba461a